src/components/betting-chips/BettingChip.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { observer } from "mobx-react";
+
+import styles from "./BettingChip.module.scss";
+import { useMainStore } from "../../hooks/useMainStore";
+
+export type BettingChipProps = {
+  value: string;
```

Comment:
You can explicitly set values that can be set for betting chip. I think for your case this would work